src/core/main.c

```diff
@@ -1818,6 +1818,19 @@ static void filter_args(
         }
 }
 
+static void stop_remaining_processes(ManagerObjective objective) {
```

Comment:
please don#t call this "stop". Stopping processes means SIGSTOP. Yes, bad name. but UNIX.

call it "zap" or so, or "finish". but not "stop", "terminate" or "kill", because of the SIGSTOP, SIGTERM, SIGKILL signals and this just creating more confusing.

Reply:
put down that thesaurus before somebody gets hurt!